app/views/notifications/_notifications.html.erb

```diff
@@ -1,10 +1,10 @@
-<% cache [current_user, @notifications] do %>
```

Comment:
Hopefully this isn't too pedantic but can this partial be renamed to something more descriptive like "notifications/_dropdown"? `notifications` is such a generic name and it confused me a bit when I was looking at this PR in order to figure out the best approach for the `index` page. I'd prefer a name which tells me exactly what the partial is used for and where rather than this name, which doesn't give me many clues.